docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -257,19 +262,19 @@ contract MyPaymaster is IPaymaster {
             msg.sender == BOOTLOADER_FORMAL_ADDRESS,
             "Only bootloader can call this method"
         );
-        // Continue execution if called from the bootloader.
+        // Continure execution if called from the bootloader.
         _;
     }
 
     constructor(address _erc20) {
         allowedToken = _erc20;
     }
 
-    function validateAndPayForPaymasterTransaction (
+    function validateAndPayForPaymasterTransaction(
         bytes32,
         bytes32,
         Transaction calldata _transaction
-    ) onlyBootloader external payable returns (bytes4 magic, bytes memory context) {
+    ) external payable returns (bytes4 magic, bytes memory context) {
```

Comment:
?

Reply:
fix incoming

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/465/commits/3ae8c68a33e059a1bf7fdf6f4a84ae2185722510